foundation/docs/blog/posts/2024-conference-recap.md

```diff
@@ -0,0 +1,301 @@
+
+* **11.6%** - Less than 1 year
+* **24.9%** - 1 to 3 years 
+* **20.2%** - 4 to 7 years
+* **20.2%** - 7 to 15 years
+* **23.1%** - 15+ years
+
+**Orders by Day**
+
+![PyTexas Orders By Day](/assets/images/blog/2024-conference-recap/PyTexas-2024-Ticket-Sales.PNG)
+
+A few conclusions we've made based on ticket sales this year and years prior:
+
+* You have your "regulars" who attend every year. They will take advantage of
+early bird sales.
+* Announcements drive ticket sales. We saw noticeable increases after we announced
+keynotes, tutorials, speakers, etc.
+    * The largest increase was after we released the schedule.
+* You will still sell a majority of your tickets in the month leading up to the event.
+
+## Sponsors
+
+Sponsorship in the return to in-person conferences world have been few and far
+between. We are immensely grateful our sponsors this year, 
+[Temporal Technologies](https://temporal.io) and [AppSignal](https://www.appsignal.com/)
+for their support. We also want to thank the [Python Software Foundation](https://python.org)
+for their generous grant. 
+
+Without them certain portions of our event would simply not have been possible 
+from a financial standpoint. 
+
+### Statistics
+This year we brought in **$12,750** from sponsorships and grants. We reached out
+to roughly 50 companies this year to sponsor PyTexas. When they responded, the
+response was usually:
+* We aren't sponsoring events right now.
+* You aren't big enough for us to consider.
+* We haven't finalized event plans, we'll get back to you.
+
+We publish this information not to shame companies who aren't sponsoring. We 
+understand the current economic climate. We write this so that other conference
+organizers who are experiencing the same thing don't feel like it's just them.
+It's everyone.
+
+## Finances
+
+PyTexas 2024 cost **$36,657.59** to operate. The largest budget items for us
+were venue, catering, A/V, and our after party. We sold 319 tickets of varying cost, 
+including add-ons such as shirts, tutorials, and the after party, and made **$36,545.00**
+in gross revenue (prior ot fees). To the best of our knowledge, this is the first
+year that ticket sales were able to completely fund the conference. This was possible
+due to the increase in capacity for in-person, plus the addition of a virtual ticket.
+
+All excess funds from this year will be used to fund ongoing PyTexas programs such
+as the meetup, infrastructure costs, and any future PyTexas event (such as the [PyTexas 2025 Conference](https://pytexas.org/2025)).
+
+## Speakers
+
+This was by far one of the most exciting speaker lineups we've ever had! We continue
+to get compliments on how good the talks were and how much everyone enjoyed them.
+This year we were overjoyed to invite [Lynn Root](https://www.roguelynn.com/) 
+and [Carol Willing](https://www.willingconsulting.com/) to keynote this year. Both
+gave _phenomenal_ keynotes that people kept raving about. From teaching Python,
```

Comment:
I think maybe start a new paragraph? This last sentence feels like it applies to the keynotes.

Reply:
Good call out. Done.